specification/managementgroups/ManagementGroups.Management/SubscriptionUnderManagementGroup.tsp

```diff
@@ -0,0 +1,109 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/rest";
+import "./models.tsp";
+import "./ManagementGroup.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+
+namespace Microsoft.Management;
+/**
+ * The details of subscription under management group.
+ */
+@parentResource(ManagementGroup)
+model SubscriptionUnderManagementGroup
+  is Azure.ResourceManager.ProxyResource<SubscriptionUnderManagementGroupProperties> {
+  ...ResourceNameParameter<
+    Resource = SubscriptionUnderManagementGroup,
+    KeyName = "subscriptionId",
+    SegmentName = "subscriptions",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface SubscriptionUnderManagementGroups {
+  /**
+   * Retrieves details about given subscription which is associated with the management group.
+   *
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "non-standard operations"
+  getSubscription is Extension.Read<
```

Comment:
same comment as HierarchySettings here - we should use ordinary resource operation template instead of extension.